.circleci/config.yml

```diff
@@ -420,3 +419,37 @@ workflows:
             - build
             - test
             - e2e-cli
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - 10.1.x
+    jobs:
+      # Linux jobs
+      - setup
+      - build:
+          requires:
+            - setup
+          filters:
+            branches:
+              ignore:
+              - /docs-preview/
+      - e2e-cli:
+          post-steps:
+            - store_artifacts:
```

Comment:
We can probably drop this as well for nightly runs since the artifacts should be the same each time.